code/lib/preview-api/src/modules/preview-web/render/StoryRender.ts

```diff
@@ -195,15 +196,15 @@ export class StoryRender<TRenderer extends Renderer> implements Render<TRenderer
         return;
       }
 
-      const renderStoryContext: StoryContext<TRenderer> = {
+      const renderStoryContext: StoryContext<TRenderer> = prepareContext({
```

Comment:
We don't want to prepare the context for loaders also? I would suggest we move this inside `StoryRender.storyContext()` (maybe move the part in `getCurrentContext()` in there too, I'm not sure why I didn't do that already.

Reply:
Thanks!